core/src/main/java/hudson/model/Queue.java

```diff
@@ -73,6 +73,8 @@
 import jenkins.util.SystemProperties;
 import jenkins.util.Timer;
 import hudson.triggers.SafeTimerTask;
+
+import java.util.*;
```

Comment:
Might not want to do that, especially with how Java adds new classes to this package all the time it seems.

Reply:
This is IntelliJ -- it has some threshold after which it decides
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html:
`Class count to use import with ‘*’` (`5`)
`Names count to use static import with ‘*’` (`3`)

If people strenuously object, I can adjust, but, we're talking to a computer program...

Reply:
I don't know of an official style guide, but I tend to make the setting for imports at 100 or something really large. Not a huge deal either way.